policy/scan/local_scanner.go

```diff
@@ -94,7 +94,7 @@ func WithReportType(reportType ReportType) ScannerOption {
 }
 
 func NewLocalScanner(opts ...ScannerOption) *LocalScanner {
-	runtime := providers.Coordinator.NewRuntime()
+	runtime := providers.DefaultRuntime()
```

Comment:
This function calls the exact same function `Coordinator.NewRuntime()` so, not a big change 

https://github.com/mondoohq/cnquery/blob/main/providers/defaults_shared.go#L24